db/migrate/20250915190655_add_csp_attrs_digest_column_to_user_verifications.rb

```diff
@@ -0,0 +1,8 @@
+class AddCspAttrsDigestColumnToUserVerifications < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def change
+    add_column :user_verifications, :csp_attrs_digest, :string, null: true
+    add_index :user_verifications, :csp_attrs_digest, unique: true, where: "csp_attrs_digest IS NOT NULL", algorithm: :concurrently
```

Comment:
I'm a little concerned that a unique index will cause unexpected issues. Login.gov allows multiple credentials to be issued with the same attributes, so this isn't a promise that it'll be unique

Reply:
Yeah good point